libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2765,6 +2765,10 @@ function notify:handle_rgb(red, green, blue, rate_hz) end
 ---@param tune string
 function notify:play_tune(tune) end
 
+-- desc
+---@param text string
+---@param row integer
```

Comment:
Don't feel constrained to the crap descriptions most of our bindings have. 
```suggestion
-- Display text on a notify display, text too long to fit will automatically be scrolled.
---@param text string -- upto 50 characters
---@param row integer -- row number to display on, 0 is at the top.
```

Reply:
Yes! Love it! Was following along